sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -188,7 +188,7 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
     }
 
     private fun setupActionBarTitle() {
-        supportActionBar?.title = uiCustomization.uiCustomization.toolbarCustomization?.headerText
+        supportActionBar?.title = uiCustomization.uiCustomizationMap[com.netcetera.threeds.sdk.api.ui.logic.UiCustomization.UiCustomizationType.DEFAULT]?.toolbarCustomization?.headerText
```

Comment:
Why does the key of `uiCustomizationMap` needs to that long?

Reply:
What about

```
import com.netcetera.threeds.sdk.api.ui.logic.UiCustomization.UiCustomizationType.DEFAULT as defaultUICustomization
```

or something like it?